<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="speedSlider">
    <property name="geometry">
     <rect>
      <x>347</x>
      <y>400</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="sliderPosition">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="stepSpinBox3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>120</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="stepSpinBox2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>120</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>377</x>
      <y>480</y>
      <width>88</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>DÃ©marrer</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="stepSpinBox1">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonUp">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>200</y>
      <width>88</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>haut</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonLeft">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>88</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>gauche</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRight">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>230</y>
      <width>88</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>droite</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDown">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>270</y>
      <width>88</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>bas</string>
    </property>
   </widget>
   <widget class="QLabel" name="speedLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>350</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
